links.html

```diff
@@ -20,14 +21,18 @@
     </header>
    
     <main>
-    <section id = "useful-links">
+    <section id="useful-links">
     
-    <h1>Useful Links</h1><div>
+    <h1>Useful Links</h1>
+    <div>
     
-   	<ul><li><a href="https://validator.w3.org/nu/" target="_blank">HTML Validator</a></li></ul>
-   	<ul><li><a href="https://jigsaw.w3.org/css-validator/" target="blamk">CSS validator</a></li></ul>
-   	<ul><li><a href="https://developer.mozilla.org/en-US/" target="blamk">MDN</a></li></ul>
-   	<ul><li><a href="https://caniuse.com/" target="blank">caniuse.com </a></li></ul></div>
+       	<ul>
+            <li><a href="https://validator.w3.org/nu/" target="_blank">HTML Validator</a></li>
+       	    <li><a href="https://jigsaw.w3.org/css-validator/" target="_blank">CSS validator</a></li>
+       	    <li><a href="https://developer.mozilla.org/en-US/" target="_blank">MDN</a></li>
+       	    <li><a href="https://caniuse.com/" target="_blank">caniuse.com </a></li>
+        </ul>
+    </div>
```

Comment:
Only one `<ul>` with multiple `<li>` items needed here
Typos in the target attribute